extensions/amp-subscriptions/0.1/viewer-subscription-platform.js

```diff
@@ -50,6 +50,8 @@ export class ViewerSubscriptionPlatform {
 
     /** @const @private {!../../../src/service/viewer-impl.Viewer} */
     this.viewer_ = Services.viewerForDoc(this.ampdoc_);
+    this.viewer_.onMessage('subscriptionchange',
```

Comment:
In viewer code we so far are using "auth" terminology. E.g. "auth", 'auth-rejected', "authorization", etc. I think we should call this similarly in this case. Maybe "auth-changed"? Or "reauth"?

Otherwise, this is good. Just needs tests.

Reply:
I used `subscriptionchange` because to match the `visibilitychange` naming convention used elsewhere.  I can change it but I'd like to retain the subscription prefix.

Re tests I'm not really sure what to test I was looking at it - they only obvious thing would be to mock the `onMessage` and then call the callback and see if `resetPlatforms()` gets called. I'll add that.  Do you see any other obvious tests?

Reply:
Ok. Sounds good.

RE: tests - yeah, this is a pretty simple one.